README.md

```diff
@@ -38,6 +38,8 @@ const loggedInUser = {
 }
 ```
 
+NOTE: email, phoneNumber and dob must match a patient or app.js will throw startup errors. 
```

Comment:
According to the [API docs](https://api.getlabs.com/docs/#getlabs-api-patient) "The patient's email, date of birth and birth sex are used to look up the patient."